launch/planning_context.launch

```diff
@@ -4,13 +4,13 @@
 
   <!-- The name of the parameter under which the URDF is loaded -->
   <arg name="robot_description" default="robot_description"/>
-  <arg name="version" default="V40"/>
 
   <!-- Load universal robot description format (URDF) -->
-  <param if="$(arg load_robot_description)" name="$(arg robot_description)" command="$(find xacro)/xacro.py '$(find nao_description)/urdf/nao$(arg version)_generated_urdf/nao_robot.xacro'"/>
-  <!-- The semantic description that corresponds to the URDF -->
-  <param name="$(arg robot_description)_semantic" textfile="$(find nao_moveit_config)/config/nao_robot_v4_H25.srdf" />
+  <param if="$(arg load_robot_description)" name="$(arg robot_description)" textfile="$(find nao_description)/urdf/naoV40_generated_urdf/nao.urdf"/>
```

Comment:
it would be awesome to keep the robot description with xacro call and the version argument (as it was previously). This way moveit will keep working for all nao models and not only nao V40.